src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.cs

```diff
@@ -26,7 +26,7 @@ public InnerPlayerControl(ILogger<InnerPlayerControl> logger, IServiceProvider s
             _eventManager = eventManager;
             _game = game;
 
-            Physics = ActivatorUtilities.CreateInstance<InnerPlayerPhysics>(serviceProvider, this);
+            Physics = ActivatorUtilities.CreateInstance<InnerPlayerPhysics>(serviceProvider, this, _eventManager, _game);
```

Comment:
Why is _eventManager here? It is in the serviceProvider

Reply:
I didn't know how it actually works.

Reply:
resolved https://github.com/Impostor/Impostor/pull/197/commits/e212d8c020c3d0097c7b696570d369d7124489c4